src/main/java/com/example/moviemanager/service/MovieService.java

```diff
@@ -49,4 +64,11 @@ public Movie update(UUID id, Movie movie) {
 
         return movieRepository.save(movie);
     }
-}
\ No newline at end of file
+
+    private MovieDetails retrieveMovieDetails(MovieDetailsResponse movieDetailsResponse) {
```

Comment:
No test for it

Reply:
created additional test, for covering cases when MovieDetailsResponse is empty and when contains more than one movie.